src/components/MessagesList/MessagesList.vue

```diff
@@ -239,6 +239,9 @@ export default {
 	beforeDestroy() {
 		EventBus.$off('scrollChatToBottom', this.handleScrollChatToBottomEvent)
 		this.cancelLookForNewMessages()
```

Comment:
```suggestion
		if (this.cancelLookForNewMessages) {
			this.cancelLookForNewMessages()
		}
```
No?

Reply:
It should not be needed, as by default [`cancelLookForNewMessages` is initialized to an empty function](https://github.com/nextcloud/spreed/blob/bbd0277f1057fb5c2cbe5431349c43cbfa6eec18/src/components/MessagesList/MessagesList.vue#L108), and any further assignment is an [Axios CancelToken source](https://github.com/nextcloud/spreed/blob/bbd0277f1057fb5c2cbe5431349c43cbfa6eec18/src/components/MessagesList/MessagesList.vue#L451). It is set to `null` only in the line after that one and, if I am not mistaken, `beforeDestroy` will be called just once in the lifecycle of a Vue component.